core/vm/opcodes.go

```diff
@@ -98,7 +98,7 @@ const (
 	TIMESTAMP   OpCode = 0x42
 	NUMBER      OpCode = 0x43
 	DIFFICULTY  OpCode = 0x44
-	RANDOM      OpCode = 0x44 // Same as DIFFICULTY
```

Comment:
Removing `RANDOM` is an API change we cannot make. You can simply add `PREVRANDAO` as an alternative name of opcode `0x44` instead.

Reply:
Not sure I get it, `RANDOM` is `PREVRANDAO`, the `DIFFICULTY` opcode is the one that we use pre-merge

Reply:
I am only looking at this from the Go package perspective. The symbol `RANDOM` is currently exported by package `vm`, but with your change, it is not present anymore. Such API changes are not allowed because there could be third-party code depending on the presence of this symbol.

Reply:
I see, but the `RANDOM` is not a symbol that we should expose

Reply:
From an API perspective, we're bound to :)
But  as Felix says, you can just define them both.